_translator-files/po/fr/Running-a-Server.po

```diff
@@ -251,92 +247,87 @@ msgstr "_Pour mettre à niveau votre serveur, répétez simplement les étapes c
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:84
-#, fuzzy
 msgid "_To amend your Server configuration, just repeat the last two steps above._"
-msgstr "_Pour mettre à niveau votre serveur, répétez simplement les étapes ci-dessus._"
+msgstr "_Pour modifier la configuration de votre serveur, il suffit de répéter les deux dernières étapes ci-dessus._"
 
 #. type: Title ##
 #: ../wiki/en/Running-a-Server.md:87
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Server Modes"
 msgstr "Types de serveur"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:90
-#, fuzzy
 msgid "Servers can be run in one of three modes (either at home or on a 3rd party host), depending on your needs."
-msgstr "Vous pouvez exécuter votre serveur de différentes manières (soit à la maison, soit sur un hébergeur tiers) :"
+msgstr "Les serveurs peuvent fonctionner selon l'un des trois modes suivants (à domicile ou sur un hôte tiers), en fonction de vos besoins."
 
 #. type: Title ###
 #: ../wiki/en/Running-a-Server.md:91
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Unregistered mode"
-msgstr "2. Non enregistré"
```

Comment:
Probably this change also breaks some html anchors. 

This isn't specific for one language but all. 
Just saying to make translators aware.

Reply:
I'm not sure what you're referring to here.
I am/was under the impression that I "fixed" a wrong translation here.
That said, is seem you're meaning some deeper things.
Could you please elaborate?

Reply:
@ann0see I had a quick look but couldn't find any anchors pointing to "Unregistered mode". In any case, if the anchor needs fixing in EN, that will make the string a fuzzy in all languages. Some anchors will get fixed in #899 though.

@trebmuh don't worry, it's not specific to FR and might not be an issue anyway.